config.py

```diff
@@ -109,6 +110,8 @@ class DevelopmentConfig(Config):
     PARTY_USERNAME = os.getenv('PARTY_USERNAME', 'admin')
     PARTY_PASSWORD = os.getenv('PARTY_PASSWORD', 'secret')
     PARTY_AUTH = (PARTY_USERNAME, PARTY_PASSWORD)
+    PARTY_FAKE_DATA = bool(os.getenv('FAKE_PARTY_DATA', False))
```

Comment:
I'm not a fan of testing parameters making their way into production. Is there some way we can confine this only to the testing code?

Reply:
It's only in the `DevelopmentConfig` and `TestingConfig` classes - as are many other similar things, like the lines above.  I kind of agree with you, but the paradigm is already set that way I'd say.

I think there would be value in adding some safeguards to this though, to prevent setting of the flags that enable this in production and pre-prod?  Maybe a check for env vars that would only be set on a live instance?

Reply:
we should refactor so that real implementations can be swapped out with mock implementations, for testing, and separate the production config from the testing config

Reply:
Done. On it's way in